egs/dirha_wsj/asr1/local/dirha_data_prep.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+# DIRHA DATA PREPARATION   (Author: Mirco Ravanelli)
```

Comment:
Is this file OK to copy to ESPnet because there is no license notice?
Maybe we should ask Mirco (I can do it if you want me to do).

Reply:
All the files in ./local are directly used or modified (for e2e setup) from https://github.com/SHINE-FBK/DIRHA_English_wsj , maybe it is better for you to ask Mirco? Thanks!

Reply:
I confirmed from Mirco that this is no problem. I put his reply as a record
![image](https://user-images.githubusercontent.com/11741550/64734892-82caab00-d4b5-11e9-8955-39c2c496e5f3.png)